app/controllers/stories/feeds_controller.rb

```diff
@@ -35,14 +35,20 @@ def assign_feed_stories
     end
 
     def signed_in_base_feed
-      strategy = AbExperiment.get(experiment: :feed_strategy_round_2, controller: self, user: current_user,
-                                  default_value: "original")
-      feed = if strategy.weighted_query_strategy?
-               Articles::Feeds::WeightedQueryStrategy.new(user: current_user, page: @page, tags: params[:tag])
-             elsif Settings::UserExperience.feed_strategy == "basic"
+      feed = if Settings::UserExperience.feed_strategy == "basic"
                Articles::Feeds::Basic.new(user: current_user, page: @page, tag: params[:tag])
              else
-               Articles::Feeds::LargeForemExperimental.new(user: current_user, page: @page, tag: params[:tag])
+               strategy = AbExperiment.get(
+                 experiment: :feed_strategy_round_2,
+                 controller: self, user: current_user,
+                 default_value: "original"
```

Comment:
Would it make sense to create a constant for "original"?  It's appearing in a few different places as a "magic value".

Reply:
Done